openshift/flower/s2i/bin/run

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+source /opt/app-root/etc/generate_container_user
+
+set -e
+
+>&2 echo "$(/usr/bin/env)"
+>&2 echo ""
+>&2 echo "PWD=$(pwd)"
+>&2 echo ""
+>&2 echo "$(ls -al)"
+>&2 echo ""
+
+function is_gunicorn_installed() {
```

Comment:
Seems like this isn't necessary. Do we need all the django/gunicorn related functions?